runtime_builders/data_integrity.py

```diff
@@ -59,42 +59,54 @@ def _verify(directory):
                       .format(ke))
 
 
-def _verify_files(staging_path, tagged_path):
-    bucket_name = staging_path.replace(GCS_FILE_PREFIX, '').split('/')[0]
-    if bucket_name not in tagged_path:
-        logging.error('Buckets do not match!')
-        logging.error('{0} || {1}'.format(staging_path, tagged_path))
-        return 1
-
+def _verify_latest_files_match(project_name, config_latest):
+    """
+    Verify that the file pointed to by <project_name>.version is the same
+    as the file specified in the builder config
+    """
+    remote_version = builder_util.RUNTIME_BUCKET_PREFIX + \
+        project_name + '.version'
     try:
         tmpdir = tempfile.mkdtemp()
-        tmp1 = os.path.join(tmpdir, 'tmp1.yaml')
-        tmp2 = os.path.join(tmpdir, 'tmp2.yaml')
-
-        _get_file_from_gcs(staging_path, tmp1)
-        _get_file_from_gcs(tagged_path, tmp2)
-
-        if not filecmp.cmp(tmp1, tmp2):
-            logging.error('Files {0} and {1} do not match!'
-                          .format(staging_path, tagged_path))
-            with open(tmp1, 'r') as f:
-                logging.error('\n' + staging_path + '\n' + f.read())
-            with open(tmp2, 'r') as f:
-                logging.error('\n' + tagged_path + '\n' + f.read())
-            return 1
+        version_file = os.path.join(tmpdir, 'runtime.version')
+        builder_util._get_file_from_gcs(remote_version, version_file)
+
+        with open(version_file, 'r') as f:
+            version_contents = f.read().strip('\n').strip(' ')
+            version_latest = builder_util.RUNTIME_BUCKET_PREFIX + \
+                project_name + '-' + version_contents
+            if version_latest != config_latest:
+                logging.error('Builders do not match!')
+                logging.error('runtime.version: {0}'.format(version_latest))
```

Comment:
Could you make these a little more clear? I'd expect output like:

Expected latest to container: %s
It contained: %s!